### PR TITLE
Rename main activity to login activity

### PR DESCRIPTION
**Pull request by manuelfalonso**

### CODE REVIEW

**Review by JonathanJusid — APPROVED** [review]
good!

File: .idea/misc.xml [hunk 1]
```diff
@@ -3,6 +3,8 @@
   <component name="DesignSurface">
     <option name="filePathToZoomLevelMap">
       <map>
+        <entry key="..\:/ORT/4to Cuatrimestre 2021/CocinArte/app/src/main/res/layout/account_profile_fragment.xml" value="0.35260416666666666" />
```
Comment (Alaple) [inline .idea/misc.xml:6]: Please delete this file and add this folder to gitignore

**Review by Alaple — CHANGES_REQUESTED** [review]
Please fix this first before merge

**Review by Alaple — APPROVED** [review]
LGTM

**Pull request merged**